myhpom/templates/myhpom/upload/requirements.html

```diff
@@ -14,57 +14,57 @@
-				<label class="upload-requirements__label" for="id_requirement_{{ requirement.id }}">{{ requirement.text }}</label>
-				<ol class="upload-requirements-link__ol">
-					{% for link in requirement.staterequirementlink_set.all %}
-					<li class="upload-requirements-link__li">
-						<a href="{{ link.href }}" onClick="event.preventDefault();">{{ link.text }}</a>
-					</li>
-					{% endfor %}
-				</ol>
-			</li>
-			{% endfor %}
-		</ol>		
-	</div>
+    <div class="col">
+        <h3 class="upload-requirements__title">Advance Directive Document Requirements for {{ user.userdetails.state.title }}</h3>
+        <ol class="upload-requirements__list">
+            {% for requirement in requirements %}
+            <li class="upload-requirements__item">
+                <input class="upload-requirements__checkbox" type="checkbox" id="id_requirement_{{ requirement.id }}" class="upload-requirements__checkbox"/>
+                <label class="upload-requirements__label" for="id_requirement_{{ requirement.id }}">{{ requirement.text }}</label>
+                <ol class="upload-requirements-link__list">
+                    {% for link in requirement.staterequirementlink_set.all %}
+                    <li class="upload-requirements-link__item">
+                        <a href="{{ link.href }}" onClick="event.preventDefault();">{{ link.text }}</a>
+                    </li>
+                    {% endfor %}
+                </ol>
+            </li>
+            {% endfor %}
+        </ol>
+    </div>
 </div>
 
 <div class="row">
-	<div class="col">
-		<h4 class="upload-requirements__h4">My file is valid starting:
-			<a class="upload-requirements__a--right" href="" onClick="event.preventDefault();">What is this?</a>
-		</h4>
-		<form id="id_requirements_form" class="upload-requirements__form" method="POST" action="{% url 'myhpom:upload_requirements' %}">
-			{% csrf_token %}
-			<div id="id_valid_date_div" class="upload-requirements__input-div{% if form.errors.valid_date %}--invalid{% endif %}">
-				<input id="id_valid_date" name="valid_date" data-min-year="{{ MIN_YEAR }}" data-max-year="{{ MAX_YEAR }}"{% if form.errors.valid_date %} class="upload-requirement__input--invalid"{% endif %}/>
-			</div>
-			<div id="id_valid_date_errors" class="upload-requirements__div-errors">
-				{{ form.errors.valid_date }}
-			</div>
-		</form>
-	</div>
+    <div class="col">
+        <h4 class="upload-requirements__item-head">My file is valid starting:
+            <a class="upload-requirements__help--right" href="" onClick="event.preventDefault();">What is this?</a>
+        </h4>
+        <form id="id_requirements_form" class="upload-requirements__form" method="POST" action="{% url 'myhpom:upload_requirements' %}">
+            {% csrf_token %}
+            <div id="id_valid_date_div">
+                <input id="id_valid_date" name="valid_date" data-min-year="{{ MIN_YEAR }}" data-max-year="{{ MAX_YEAR }}"/>
```

Comment:
This field is crammed together with the above/below. Wrapping it in a `p` (or some other bs4 block) would make it match the other forms margins.